open-sauced/intro
# fix(favicon): moved the favicon to /static/img in order to show up in…

## Description

This PR adds a favicon to the Docusaurus documentation app located at opensauced.pizza/learn. The favicon was missing, which affected visual identification in browser tabs and bookmarks.

## Related Tickets & Documents
Fixes #236 

## Mobile & Desktop Screenshots/Recordings

![Screenshot 2024-09-19 at 2 06 44 PM](https://github.com/user-attachments/assets/f6f8e32e-143f-4d9e-a0b4-d8d44c2b41f8)


![Screenshot 2024-09-19 at 4 14 54 PM](https://github.com/user-attachments/assets/fcc3fc3c-9880-4d09-8557-dec4d2ff45f0)


## Steps to QA
1. Pull this branch and run the Docusaurus app locally
2. Open the app in a web browser
3. Verify that the favicon appears in the browser tab
4. Check different browsers (Chrome, Firefox, Safari, etc.) to ensure consistency


## Tier (staff will fill in)


## [optional] What gif best describes this PR or how it makes you feel?

![tmnt](https://github.com/user-attachments/assets/52bf1f7e-3e17-4ecd-8c7a-912d4882a259)
